src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings

```diff
@@ -927,6 +927,20 @@ Bei ausgeschalteter Hintergrundaktualisierung müssen Sie die App täglich aufru
 
 "ExposureSubmission_OverwriteNotice_Antigen_Text" = "Sie haben bereits einen Schnelltest registriert. Die App kann maximal einen Schnelltest und einen PCR-Test gleichzeitig verwalten. Wenn Sie einen weiteren Schnelltest registrieren, wird der erste Schnelltest aus der App gelöscht.";
 
+/* ExposureSubmission AntigenTest Profile Information */
+
+"ExposureSubmission_Antigen_Profile_Information_Title" = "Schnelltest-Profil";
+
+"ExposureSubmission_Antigen_Profile_Information_Image_Description" = "Eine QR Code der eine Person auf dem Smartphone identifiziert.";
```

Comment:
Hi @KaiTeuber - I have just provided a screenreader text here for Android - could you please use the same one for iOS? Thanks
"ExposureSubmission_Antigen_Profile_Information_Image_Description" = "Eine Frau mit einem Smartphone in der Hand steht vor einem Gebäude. Ein QR-Code symbolisiert den zu scannenden Schnelltest-Profil.";
```

Reply:
Hi @Ein-Tim, I have made an alternative text proposal :-) (see my comment)